spec/integration/data_stream/manifest.spec.yml

```diff
@@ -222,5 +222,19 @@ spec:
               type: array
               items:
                 type: string
+        source_mode:
+          description: |-
+            Source mode to use. This configures how the document source (`_source`) is stored
+            for this data stream.
+            If configured as `default`, this mode is not configured and it uses Elasticsearch defaults.
```

Comment:
It would be nice to add a link to https://www.elastic.co/guide/en/elasticsearch/reference/8.4/mapping-source-field.html#synthetic-source here so anyone trying to figure out what this is can directly read up on it.

Reply:
Added :+1: